src/run-es.js

```diff
@@ -40,13 +46,13 @@ class Elastic {
     if (!this.currVer || this.newVer > this.currVer || nconf.get('es_upgrade') === true) {
       this.doUpgrade();
     } else {
-      logger.debug(` >> no upgrades needed ... ES already has v${this.env.APP_VERSION} template`);
+      logger.debug(`[Elasticsearch] - ${this.esHostPort} - already up-to-date [v${this.env.APP_VERSION}]`);
     }
   }
 
   async doUpgrade() {
     // Import/update latest visualizations and dashboards
-    logger.debug(` >> Upgrading Elasticsearch to v.${this.newVer} ... ` +
+    logger.debug(`[Elasticsearch] - ${this.esHostPort} - upgrading to v.${this.newVer} ... ` +
```

Comment:
This log format looks like something that might be worth extracting into its own method - changing the format of the log message looks painful and error prone in its current state.

Reply:
Done! Thx!